docs/src/index.md

```diff
@@ -344,6 +348,22 @@ To get in touch with the developers,
 or [create an issue](https://github.com/trixi-framework/Trixi.jl/issues/new).
 
 
+## Participating research groups
+Participating research groups in alphabetical order:
+
+[Applied and Computational Mathematics, RWTH Aachen University :de:](https://www.acom.rwth-aachen.de)
+
+[Applied Mathematics, Linköping University :sweden:](https://liu.se/en/employee/andwi94)
```

Comment:
```suggestion
[Applied and Computational Mathematics, RWTH Aachen University :de:](https://www.acom.rwth-aachen.de)

[Applied Mathematics, Department of Mathematics, University of Hamburg :de:](https://www.math.uni-hamburg.de/en/forschung/bereiche/am.html)

[Applied Mathematics, Linköping University :sweden:](https://liu.se/en/employee/andwi94)
```

Reply:
Which one of the two comments is the one you intend to be merged? This one or the earlier one?

Reply:
Both since you copied this stuff manually to two files.

Reply:
He who reads, succeeds 🤦